source/common/upstream/upstream_impl.h

```diff
@@ -72,6 +72,38 @@ class HealthCheckHostMonitorNullImpl : public HealthCheckHostMonitor {
   void setUnhealthy(UnhealthyType) override {}
 };
 
+/**
+ * Implementation of LoadMetricStats.
+ */
+class LoadMetricStatsImpl : public LoadMetricStats {
+public:
+  LoadMetricStatsImpl() = default;
```

Comment:
I think this can be removed.

Reply:
I believe I need to explicitly declare the default constructor since I've deleted the copy and assignment constructors below.